package.json

```diff
@@ -86,6 +86,7 @@
     "express": "^4.19.2",
     "express-session": "^1.18.0",
     "husky": "^9.0.11",
+    "ioredis": "^5.4.1",
```

Comment:
are you using this?

Reply:
i was about to use it but realy i'm not using it currently, since token don't last long(10min) i decided not to blacklist verification token s

Reply:
remove it then